README.md

```diff
@@ -35,10 +43,18 @@ If you wish to manually create and publish a package for Python:
     python setup.py sdist
     twine upload --repository-url ${PYPI_REPOSITORY_URL} --username ${PYPI_USER} --password ${PYPI_PASSWORD} dist/*
     ```
-    `PYPI_REPOSITORY_URL`, `PYPI_USER` and `PYPI_PASSWORD` are parameters for publishing.
+   `PYPI_REPOSITORY_URL`, `PYPI_USER` and `PYPI_PASSWORD` are parameters for publishing.
 
 ## Release notes
 
+### 4.3.0
+
++ Added golang specific properties
++ Updated bom:4.4.0
++ Updated grpc-service-generator:3.4.0
++ Updated grpcio-tools:1.54.2
```

Comment:
We actually updated to 1.56.0 if I am not mistaken. Could you please check that moment?